### DISCUSSION TITLE
Feature Request: Private Mode | Paranoiac Mode

### DISCUSSION THREAD

**Discussion by danrobi11** [minbrowser/min#1569] [Ideas]
Option to put Min in private mode.
Im aware of the private tab. But when we're clicking on a link inside another app. Min open the tab in a normal tab.
Therefore, a private mode is needed.

Also, a paranoiac mode would be neat. Which would remove everything Min has cached since day 1. Basically everything except the config file.
Thank you for min.

**Comment by lubiedo** [comment]
I think the global private mode sounds like a good idea. @PalmerAL Something like this could be used: https://github.com/lubiedo/min/commit/d7e3301db277ffcad51866eea214810ef8566651. What do you think?
